core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -912,8 +912,22 @@ ExWorkProcRetcode ExHbaseAccessUpsertVsbbSQTcb::work()
 	    copyRowIDToDirectBuffer(rowId_);
 
 	    currRowNum_++;
-	    matches_++;
-
+            // temp code below.WIll remove the envvar after this
+            //has got enough exposure in  testing.
+            if (getenv("TURN_OFF_RETURNROW")) 
```

Comment:
Please consider removing this. From what I can tell from Google, getenv will have a performance impact, Particularly when called so often (once per row?). A cqd or COMP_BOOL can be used to tun feature of temporarily. If a runtime toggle is needed, maybe we call getenv at the top somewhere and store the value in a local variable.